web/client/components/Version/VersionDialog.jsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Dialog from '../misc/Dialog';
+import { Glyphicon } from 'react-bootstrap';
+import Message from '../I18N/Message';
+import Button from '../misc/Button';
+import PropTypes from 'prop-types';
+import '../../themes/default/less/version.less';
```

Comment:
sorry, but less files needs to be imported in the theme here
https://github.com/geosolutions-it/MapStore2/blob/17c1993b4495086616edf9828df30d6f20b28751/web/client/themes/default/less/mapstore.less#L81

which is already there, so no need to import it here
```suggestion

```